app/scripts/lib/snap-keyring/keyring-snaps-permissions.test.ts

```diff
@@ -25,6 +25,37 @@ describe('keyringSnapPermissionsBuilder', () => {
     subjectType: SubjectType.Website,
   });
 
+  describe('Portfolio origin', () => {
+    it('returns the methods Portfolio can call', () => {
+      const permissions = keyringSnapPermissionsBuilder(
+        mockController,
+        'https://portfolio.metamask.io',
+      );
+      expect(permissions()).toStrictEqual([
```

Comment:
We could use `PORTFOLIO_ALLOWED_METHODS` if we export it, WDYT?

Reply:
I think it’s better to keep it hard-coded, it adds friction to change the methods.